completions/zsh/_riverctl

```diff
@@ -0,0 +1,134 @@
+    'zoom:Bump the focused view to the top of the layout stack'
+    # Tag management
+    'set-focused-tags:Show views with tags corresponding to the set bits of tags'
+    'set-view-tags:Assign the currently focused view the tags corresponding to the set bits of tags'
+    'toggle-focused-tags:Toggle visibility of views with tags corresponding to the set bits of tags'
+    'toggle-view-tags:Toggle the tags of the currently focused view'
+    'spawn-tagmask:Set a tagmask to filter the tags assigned to newly spawned views on the focused output'
+    # Mappings
+    'declare-mode:Create a new mode'
+    'enter-mode:Switch to given mode if it exits'
+    'map:Run command when key is pressed while modifiers are held down and in the specified mode'
+    'map-pointer:Move or resize views when button and modifiers are held down while in the specified mode'
+    'unmap:Remove the mapping defined by the arguments'
+    'unmap-pointer:Remove the pointer mapping defined by the arguments'
+    # Configuration
+    'attach-mode:Configure where new views should attach to the view stack for the currently focused output'
+    'background-color:Set the background color'
+    'border-color-focused:Set the border color of focused views'
+    'border-color-unfocused:Set the border color of unfocused views'
+    'border-width:Set the border width to pixels'
+    'focus-follows-cursor:There are three available modes'
+    'opacity:Configure server-side opacity of views'
+    'outer-padding:Set the padding around the edge of the screen to pixels'
+    'set-repeat:Set the keyboard repeat rate to rate key repeats and repeat delay'
+    'view-padding:Set the padding around the edge of each view to pixels'
+    'xcursor-theme:Set the xcursor theme'
+    # Options
+    'declare-option:Declare a new option with the given type and inital value'
+    'get-option:Print the current value of the given option to stdout'
+    'set-option:Set the value of the specified option to value'
+    'mod-option:Add value to the value of the specified option'
+    'output_title:Changing this option changes the title of the wayland and X11 backend outputs'
+  )
+
+  local -A opt_args
+
+  _arguments -C '*:: :->_cmds'
+
+  if (( CURRENT == 1 )); then
+    _describe -t commands "commands" _cmds
+    return
+  fi
+
+  case "$words[1]" in
+    close\
+      |csd-filter-add\
+      |exit\
+      |float-filter-add\
+      |mod-main-count\
+      |mod-main-factor\
+      |spawn\
+      |toggle-float\
+      |toggle-fullscreen\
+      |zoom\
+      |set-focused-tags\
+      |set-view-tags\
+      |toggle-focused-tags\
+      |toggle-view-tags\
+      |spawn-tagmask\
+      |decalre-mode\
+      |enter-mode\
+      |map-pointer\
+      |unmap-pointer\
+      |background-color\
+      |border-color-focused\
+      |border-color-unfocused\
+      |border-width\
+      |opacity\
+      |outer-padding\
+      |set-repeat\
+      |view-padding\
+      |xcursor-theme\
+      |outputs)
```

Comment:
I think this could be simplified with `_cmds`

Reply:
Oh yeah it works, didn't think about it, just
``` bash
  case "$words[1]" in
    _cmds) return 0 ;;
```

Reply:
> ```shell
>   case "$words[1]" in
>     _cmds) return 0 ;;
> ```

Wait, that's illegal

I would expect something like this:

```shell
case "$words[1]" in
  # snip
  *)
    return 0
  ;;
esac
```

Reply:
Yeah it's better